2023-11-19 14:20:03,900 - root - INFO - Initiating present weather
2023-11-19 14:20:03,904 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): radar.bmkg.go.id:8060
2023-11-19 14:20:03,980 - urllib3.connectionpool - DEBUG - https://radar.bmkg.go.id:8060 "GET /getRadarGeotif?token=19387e71e78522ae4172ec0fda640983b8438c9cfa0ca571623cb69d8327&radar=amandemenForecast&type=latest HTTP/1.1" 200 90
2023-11-19 14:20:03,983 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fe7b3cc75c0>
2023-11-19 14:20:03,984 - rasterio.env - DEBUG - Starting outermost env
2023-11-19 14:20:03,984 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-19 14:20:03,985 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> created
2023-11-19 14:20:03,989 - rasterio._filepath - DEBUG - Installing FilePath filesystem handler plugin...
2023-11-19 14:20:03,990 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-19 14:20:03,990 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-19 14:20:03,990 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fe7b3d3ff10>.
2023-11-19 14:20:03,990 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fe7b3cc75c0>
2023-11-19 14:20:03,991 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-19 14:20:04,177 - rasterio._base - DEBUG - Nodata success: 1, Nodata value: nan
2023-11-19 14:20:04,177 - rasterio._base - DEBUG - Dataset <open DatasetReader name='https://radar.bmkg.go.id/raster/radarsatelit_202311191410.tif' mode='r'> is started.
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fe7b3cc75c0>
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> options
2023-11-19 14:20:04,178 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10>.
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - Exiting outermost env
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fe7b3cc75c0>
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fe7b3f3e1b0>
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - Starting outermost env
2023-11-19 14:20:04,178 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-19 14:20:04,179 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> created
2023-11-19 14:20:04,179 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-19 14:20:04,179 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-19 14:20:04,179 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fe7b3d3ff10>.
2023-11-19 14:20:04,180 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fe7b3f3e1b0>
2023-11-19 14:20:04,180 - rasterio.env - DEBUG - Got a copy of environment <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> options
2023-11-19 14:20:04,180 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fe8170045c0>
2023-11-19 14:20:04,181 - rasterio.env - DEBUG - Got a copy of environment <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> options
2023-11-19 14:20:04,181 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fe8170045c0>
2023-11-19 14:20:04,181 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-19 14:20:04,182 - rasterio._base - DEBUG - Nodata success: 1, Nodata value: nan
2023-11-19 14:20:04,182 - rasterio._base - DEBUG - Dataset <open DatasetReader name='https://radar.bmkg.go.id/raster/radarsatelit_202311191410.tif' mode='r'> is started.
2023-11-19 14:20:04,182 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fe8170045c0>
2023-11-19 14:20:04,182 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> options
2023-11-19 14:20:04,182 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10>.
2023-11-19 14:20:04,182 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-19 14:20:04,182 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> created
2023-11-19 14:20:04,183 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-19 14:20:04,183 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-19 14:20:04,183 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fe7b3d3ff10>.
2023-11-19 14:20:04,183 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fe8170045c0>
2023-11-19 14:20:04,211 - rasterio._io - DEBUG - Output nodata value read from file: nan
2023-11-19 14:20:04,211 - rasterio._io - DEBUG - Output nodata values: [nan]
2023-11-19 14:20:04,212 - rasterio._io - DEBUG - all_valid: False
2023-11-19 14:20:04,212 - rasterio._io - DEBUG - mask_flags: ([<MaskFlags.nodata: 8>],)
2023-11-19 14:20:04,212 - rasterio._io - DEBUG - Jump straight to _read()
2023-11-19 14:20:04,212 - rasterio._io - DEBUG - Window: Window(col_off=0, row_off=0, width=10215, height=3792)
2023-11-19 14:20:04,213 - rasterio._io - DEBUG - IO window xoff=0.0 yoff=0.0 width=10215.0 height=3792.0
2023-11-19 14:20:04,855 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fe7b3f3e1b0>
2023-11-19 14:20:04,855 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10> options
2023-11-19 14:20:04,855 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fe7b3d3ff10>.
2023-11-19 14:20:04,855 - rasterio.env - DEBUG - Exiting outermost env
2023-11-19 14:20:04,855 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fe7b3f3e1b0>
2023-11-19 14:20:04,857 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): radar.bmkg.go.id:8060
2023-11-19 14:20:16,732 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fe7b3d02c30>
2023-11-19 14:20:16,733 - rasterio.env - DEBUG - Starting outermost env
2023-11-19 14:20:16,733 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-19 14:20:16,733 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fe7b3d75b70> created
2023-11-19 14:20:16,734 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-19 14:20:16,734 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-19 14:20:16,734 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fe7b3d75b70>.
2023-11-19 14:20:16,734 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fe7b3d02c30>
2023-11-19 14:20:16,734 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-19 14:20:16,736 - rasterio._base - DEBUG - Nodata success: 0, Nodata value: 0.000000
2023-11-19 14:20:16,736 - rasterio._base - DEBUG - Dataset <open DatasetReader name='/home/swx/present_weather/src/H09_B13_Indonesia_20231119.Z1410.cor.tiff' mode='r'> is started.
2023-11-19 14:20:16,736 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fe7b3d02c30>
2023-11-19 14:20:16,736 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fe7b3d75b70> options
2023-11-19 14:20:16,736 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fe7b3d75b70>.
2023-11-19 14:20:16,736 - rasterio.env - DEBUG - Exiting outermost env
2023-11-19 14:20:16,736 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fe7b3d02c30>
2023-11-19 14:20:16,737 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fe7b3739310>
2023-11-19 14:20:16,737 - rasterio.env - DEBUG - Starting outermost env
2023-11-19 14:20:16,737 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-19 14:20:16,737 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fe7b37394e0> created
2023-11-19 14:20:16,737 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-19 14:20:16,738 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-19 14:20:16,738 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fe7b37394e0>.
2023-11-19 14:20:16,738 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fe7b3739310>
2023-11-19 14:20:16,738 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-19 14:20:16,739 - rasterio._base - DEBUG - Nodata success: 0, Nodata value: 0.000000
2023-11-19 14:20:16,739 - rasterio._base - DEBUG - Dataset <open DatasetReader name='/home/swx/present_weather/src/H09_B13_Indonesia_20231119.Z1410.cor.tiff' mode='r'> is started.
2023-11-19 14:20:16,739 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fe7b3739310>
2023-11-19 14:20:16,739 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fe7b37394e0> options
2023-11-19 14:20:16,739 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fe7b37394e0>.
2023-11-19 14:20:16,739 - rasterio.env - DEBUG - Exiting outermost env
2023-11-19 14:20:16,740 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fe7b3739310>
2023-11-19 14:20:16,754 - asyncio - DEBUG - Using selector: EpollSelector
2023-11-19 14:20:20,104 - root - INFO - dask.bag<lambda, npartitions=34>
2023-11-19 14:21:04,998 - root - INFO - Total elapsed time: 0:01:01.099502
